docs/fundamentals/productivity/code-analysis.md

```diff
@@ -35,11 +40,24 @@ The following rules are enabled, by default, in .NET 5.0 Preview 8.
 | [CA2015](/visualstudio/code-quality/ca2015) | Reliability | Warning | Do not define finalizers for types derived from <xref:System.Buffers.MemoryManager%601> |
 | [CA2247](/visualstudio/code-quality/ca2247) | Usage | Warning | Argument passed to TaskCompletionSource constructor should be <xref:System.Threading.Tasks.TaskCreationOptions> enum instead of <xref:System.Threading.Tasks.TaskContinuationOptions> |
 
-You can change the severity of these rules to disable them or elevate them to errors. In addition, you may want to enable additional rules that are included in this release but are disabled by default. For more information, see [Configure code analysis rules](configure-code-analysis-rules.md).
+You can change the severity of these rules to disable them or elevate them to errors.
+
+### Enabling additional rules
```

Comment:
```suggestion
### Enable additional rules
```

Reply:
Doesn't most of this section apply to code style rules too? If so, we could break it out into an H2 after the code style rules section.

Reply:
No, it only applies to CA rules.